src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -1674,40 +1674,43 @@ public void Downing(Address address)
             var localGossip = LatestGossip;
             var localMembers = localGossip.Members;
             var localOverview = localGossip.Overview;
-            var localSeen = localOverview.Seen;
             var localReachability = _membershipState.DcReachability;
 
             // check if the node to DOWN is in the 'members' set
-            var member = localMembers.FirstOrDefault(m => m.Address == address);
-            if (member != null && member.Status != MemberStatus.Down)
+            var found = false;
+            foreach (var member in localMembers.Where(m => m.Address == address))
```

Comment:
Found the same type of issue with the `Downing` function too and made similar fixes there.